stackdriver_debugger.c

```diff
@@ -325,11 +317,15 @@ PHP_FUNCTION(stackdriver_debugger_add_snapshot)
         }
     }
 
-    if (current_file == NULL) {
-        current_file = EX(prev_execute_data)->func->op_array.filename;
-    }
+    if (source_root == NULL) {
+        source_root = zend_string_dup(EX(prev_execute_data)->func->op_array.filename, 1);
```

Comment:
It seems like `zend_string_dup` returns the original string when it's interned. If we're sure the `op_array.filename` is always not interned, it's safe though.

Reply:
Good point, so far in testing this hasn't been the case, but we can make sure. In this case, I'll actually dup the char*